resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psd1

```diff
@@ -12,19 +12,21 @@
     DscResourcesToExport = @(
         'Text',       
         'Magnifier',
-        'MousePointer'   
+        'MousePointer',   
+        'TransparencyEffects'

```

Comment:
After looking at the other PR, I am wondering if all of these DSC resources that you're adding should be grouped under `VisualEffects` since that is what I see when I open up the settings menu.

Reply:
This seems like a good idea, as the general goal is to capture all [Accessibility](https://github.com/microsoft/winget-dsc/issues/53) settings, then eventually ALL settings, in this way. 
- What would be the best way to structure this? 
- What is the syntax to group items in a list like this one?